src/context/yaml/index.ts

```diff
@@ -69,6 +69,20 @@ export default class YAMLContext {
       }
     }
 
+    const excludedAssetsFiltered = Object.keys(this.assets).reduce(
+      (acc: Assets, key: AssetTypes) => {
+        const excludedAssetTypes = this.config.AUTH0_EXCLUDED || [];
+        if (excludedAssetTypes.includes(key)) return acc;
+
+        return {
+          ...acc,
+          [key]: this.assets[key],
+        };
+      },
+      {}
+    );
+    this.assets = excludedAssetsFiltered;
```

Comment:
The YAML context handler is a bit different than the directory context handler. It's not excluding the handlers from the batch but instead plucks off the individual values from the main YAML file. The end result is the same, but just the main technical difference between the two context handlers.